src/common/SPELLS/shadowlands/legendaries/index.ts

```diff
@@ -12,7 +11,7 @@ import SHAMAN from './shaman';
 import WARLOCK from './warlock';
 import WARRIOR from './warrior';
 
-const legendaries = spellIndexableList({
+const legendaries = {
```

Comment:
you should apply the `legendarySpellIndexableList` here, instead of in the per-class list.